Dockerfile-onbuild.template

```diff
@@ -1,4 +1,4 @@
-FROM node:0.0.0-stretch
+FROM node:0.0.0-buster
```

Comment:
Drop this change please. This PR should not change the default versions.

Also note that the onbuild variant is deprecated, so there's not much value in changing this anyway.